python/pyarrow/_s3fs.pyx

```diff
@@ -157,11 +157,16 @@ cdef class S3FileSystem(FileSystem):
                                         'port': 8020, 'username': 'username',
                                         'password': 'password'})
     allow_bucket_creation : bool, default False
-        Whether to allow CreateDir at the bucket-level. This option may also be 
+        Whether to allow CreateDir at the bucket-level. This option may also be
         passed in a URI query parameter.
     allow_bucket_deletion : bool, default False
-        Whether to allow DeleteDir at the bucket-level. This option may also be 
+        Whether to allow DeleteDir at the bucket-level. This option may also be
         passed in a URI query parameter.
+    retry_strategy : str, default None
+        The name of the retry strategy to use with S3.  Valid values are
+        "aws_standard" and "aws_default". Ignore with None.
```

Comment:
Or otherwise maybe just word as "If not given, uses a default retry strategy" or something similar (don't know if it's worth describing the 'legacy' retry strategy)

Reply:
Talking about "default" here would introduce confusion with "aws_default" (the AWS SDK's fault for poor naming, between "standard", "default" and "legacy"...)